src/style.css

```
@@ -1,3 +1,83 @@
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+
+  --blue-color: #00ACDC;
+}
+
+
+.header {
+  box-sizing: border-box;
+  width: 100vw;
+  padding: 0 50px;
+  background-color: #fff;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+}
+
+.nav_link {
+  font-size: 12px;
+  font-weight: 500;
+  text-decoration: none;
+  color: #000;
+  text-transform: uppercase;
+
+  margin: 0 10px;
+  height: 60px;
+
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  justify-content: center;
+  align-items: stretch;
+}
+
+.nav_link:hover {
+  color: var(--blue-color);
+}
+
+
+.logo_link {
+  display: flex;
+  align-items: center;
+  flex-grow: 1;
+}
+
+.is-active {
+  color: var(--blue-color);
+  justify-content: space-between;
+}
```

it's better to add styles more specific
```suggestion
.nav_link.is-active {
  color: var(--blue-color);
}
```
you don't need justify-content -> you doesn't use flex here and it's not needed (without `display: flex` this property do nothing :wink: )

---

That means i should also add `display: flex` in `.nav_link.is-active`? `.nav_link` alreadey has `display: flex`

---

After the changes i dont't need justify-content here